tests/core/util/test_config.py

```diff
@@ -313,3 +303,34 @@ async def test_non_atomic_writes(self, root_path_populated_with_config, default_
                         )
                     )
             await asyncio.gather(*all_tasks)
+
+    @pytest.mark.parametrize("prefix", [None])
```

Comment:
I'm iffy about this passing of parameters to the fixtures like this.  It also seems like a single fixture doing too much.  But, for at least some of this I'll need to dig into it some other time.